### ISSUE TITLE
Update dataset document

### ISSUE THREAD

**Issue by M3nin0** [brazil-data-cube/bdc-odc#10]
With BDC product updates it will be necessary to update the way some elements are recovered. For example, the asset bands

**Comment by M3nin0** [comment]
The new document's pattern are

```
"band2": {
        ...
        "id": 7,
        "name": "band2",
        "common_name": "blue",
        ...
}  
```

**Comment by M3nin0** [comment]
Now stac2odc accept STAC 0.8 and 0.9! fixed in #12
